### PR TITLE
Jal/readme update 01 24

### PR DESCRIPTION
**Pull request by jleslie85**
Please review 
(1) support notice and comments that indicate what's coming later this year. 
(2) also added to images, nuodb logo and Insights to dress up the site a bit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ A Kubernetes Operator written in Golang that automates the packaging, provisioni
 * Rancher Kubernetes Manager
   * Rancher RKE and Rancher supported Kubernetes (e.g. EKS, AKS) on Rancher supported cloud platforms
 
+**Note:** The NuoDB Operator is ideal for development, test and evaluation deployments. The NuoDB Operator is not yet available for production use. Coming soon in 1H'20 is automated LEVEL 2 (rolling upgrade) and LeVEL 3 (backup and recovery) operational cabability in a single cluster environment. Multi-cluster support is on the roadmap for 2H'20.
```
Comment (vegichan) [inline README.md]: LeVEL has a weird case

Comment (jleslie85) [reply README.md]: Thanks for finding that small "e" typo. Fixed.

File: README.md [hunk 2]
```diff
@@ -183,6 +187,8 @@ oc adm policy add-scc-to-user privileged system:serviceaccount:$OPERATOR_NAMESPA
 
 ## Configure NuoDB Insights Visual Monitor
 
+![](https://www.nuodb.com/sites/default/files/nuodb-insights.png)
```
Comment (vegichan) [inline README.md]: if you want to include images, they should live in this repo rather than depending on the NuoDB page.

Comment (jleslie85) [reply README.md]: Can we do this at a later time? For now I'm picking up our latest images from our website.

File: README.md (hunk 1 above)
Comment (jleslie85) [reply README.md]: I also removed "for 2H'20' to leave MC support a bit more open.

**Comment by jleslie85** [comment]
@vegichan  and @rshaull  any additional questions, comments, suggestions on this PR? is it ready for your approval? Thanks.

**Comment by jleslie85** [comment]
@vegichan and @rshaull I just added a few more improvements that help users walk-thru themselves the steps to evaluate scale-out and continuously availability. They can also use the video content to assist with this as well.

**Pull request merged**